### PR TITLE
Feature bot 1427 migration python3

### PR DESCRIPTION
**Pull request by kun2985**
# Description

This PR migrate chaos to python3

## Issue (optional)

Issue link: BOT-1427

## How to test

    python3 -m venv venv

or 

    virtualenv -p python3 venv

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,37 +1,54 @@
...
+kombu==4.6.4
+lettuce==0.2.23
+linecache2==1.0.0
+Mako==1.1.0
+MarkupSafe==1.1.1
+mock==3.0.5
+more-itertools==7.2.0
+nose==1.3.7
+pbr==5.4.3
+pkg-resources==0.0.0
```
Comment (Fahmus) [inline requirements.txt]: not in use

File: requirements.txt [hunk 2]
```diff
@@ -1,37 +1,54 @@
...
-redis==2.9.1
-requests==2.20.0
+jsonschema==3.0.2
+kombu==4.6.4
+lettuce==0.2.23
+linecache2==1.0.0
+Mako==1.1.0
+MarkupSafe==1.1.1
+mock==3.0.5
+more-itertools==7.2.0
```
Comment (Fahmus) [inline requirements.txt]: not in use

File: setup.py [hunk 3]
```diff
@@ -17,7 +17,7 @@ def finalize_options(self):
         pass
 
     def run(self):
-        os.system('{protoc} -Iproto --python_out=chaos proto/*.proto'.format(protoc=protoc))
+        os.system('{protoc} -Iproto --python_out=. proto/*.proto'.format(protoc=protoc))
```
Comment (Fahmus) [inline setup.py]: There is a reason to change python_out ?

**Comment by pthegner** [comment]
We close this PR (it's more a POC than anything because we can't publish / use it for now)

**Pull request closed without merging**